diff --git a/ros2/rcl_interfaces/action_msgs/CMakeLists.txt b/ros2/rcl_interfaces/action_msgs/CMakeLists.txt
index b5f34b9..65db062 100644
--- a/ros2/rcl_interfaces/action_msgs/CMakeLists.txt
+++ b/ros2/rcl_interfaces/action_msgs/CMakeLists.txt
@@ -15,7 +15,7 @@ find_package(ament_cmake REQUIRED)
 find_package(builtin_interfaces REQUIRED)
 # Depend on "core" generators instead of "default" generators
 # because ROS actions depend on this package
-find_package(rosidl_core_generators REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
 find_package(unique_identifier_msgs REQUIRED)
 
 set(msg_files
diff --git a/ros2/rcl_interfaces/builtin_interfaces/CMakeLists.txt b/ros2/rcl_interfaces/builtin_interfaces/CMakeLists.txt
index 282d38e..67c1339 100644
--- a/ros2/rcl_interfaces/builtin_interfaces/CMakeLists.txt
+++ b/ros2/rcl_interfaces/builtin_interfaces/CMakeLists.txt
@@ -14,7 +14,7 @@ endif()
 find_package(ament_cmake REQUIRED)
 # Depend on "core" generators instead of "default" generators
 # because ROS actions depend on this package
-find_package(rosidl_core_generators REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
diff --git a/ros2/rcl_interfaces/type_description_interfaces/CMakeLists.txt b/ros2/rcl_interfaces/type_description_interfaces/CMakeLists.txt
index c3ac72b..4374456 100644
--- a/ros2/rcl_interfaces/type_description_interfaces/CMakeLists.txt
+++ b/ros2/rcl_interfaces/type_description_interfaces/CMakeLists.txt
@@ -13,7 +13,7 @@ endif()
 find_package(ament_cmake REQUIRED)
 # Depend on "core" generators instead of "default" generators
 # because other generators depend on this.
-find_package(rosidl_core_generators REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
diff --git a/ros2/unique_identifier_msgs/CMakeLists.txt b/ros2/unique_identifier_msgs/CMakeLists.txt
index 382b8ec..00498c3 100644
--- a/ros2/unique_identifier_msgs/CMakeLists.txt
+++ b/ros2/unique_identifier_msgs/CMakeLists.txt
@@ -15,7 +15,7 @@ endif()
 find_package(ament_cmake REQUIRED)
 # Depend on "core" generators instead of "default" generators
 # because ROS actions depend on this package
-find_package(rosidl_core_generators REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
